### ISSUE TITLE
Mica For Everyone not opening 

### ISSUE THREAD

**Issue by B-Love10** [MicaForEveryone/MicaForEveryone#381]
Hi,

When I try to install mica, everything goes fine till I got a message when I try to run the application,
''To run this application, you must install .NET core. Would you like to download it now ?''

I click yes, then nothing happen. 

Tried to run again and same message, tried download and install succesfully .NET Core manually and same message. 

Any clue ? 

Thanks

**Comment by dongle-the-gadget** [comment]
https://dotnet.microsoft.com/download/dotnet/3.1

**Comment by queenbiscuit311** [comment]
Had this same issue. Would probably be best to put a link to in the readme if possible

**Comment by Asad-13** [comment]
> https://dotnet.microsoft.com/download/dotnet/3.1

The website says that support for that version has been dropped. Will it cause problems if I download a newer version?

Also, a more fundamental question perhaps, will this software being deprecated cause any security issues on my device? I'm sorry if I sound clueless, because on the topic of .NET, I practically am.

**Comment by dongle-the-gadget** [comment]
> Will it cause problems if I download a newer version

Yes, newer major versions of .NET Core isn't backwards-compatible with old ones.

> Will this software being deprecated cause any security issues on my device

Could be, I'm not sure.

**Comment by queenbiscuit311** [comment]
> > https://dotnet.microsoft.com/download/dotnet/3.1
> 
> The website says that support for that version has been dropped. Will it cause problems if I download a newer version?
> 
> Also, a more fundamental question perhaps, will this software being deprecated cause any security issues on my device? I'm sorry if I sound clueless, because on the topic of .NET, I practically am.

Not sure about the security part, but if my information is right having to download dozens of different variations of old dependencies is par for the course for windows software due to the fact that versions in programs like .NET and visual c++ are not backwards compatible. There's a good chance a lot of old versions of software like .NET are already installed on your system.
